### PR TITLE
Fix: some consistency and more...

### PR DESCRIPTION
**Pull request by arHSM**
Some consistency and a believable fake token 😉 based on your snowflake 😳 .

Also I assume you wanted to put this https://github.com/Bluenix2/wumpy/blob/main/library/wumpy/gateway/shard.py and not this https://github.com/Bluenix2/wumpy/library/wumpy-gateway/wumpy/gateway/shard.py ?

The token generation is as simple as this
```py
import base64
import hashlib
import hmac

DISCORD_TOKEN_EPOCH = 1293840000000

def generate(version: int, snowflake: int, secret: str, timestamp: int) -> str:
    parts = [
        base64.b64encode(str(snowflake).encode()),
        base64.b64encode(str(timestamp-(DISCORD_TOKEN_EPOCH/1000)).encode()),
    ]
    t_hmac = hmac.new(str(secret).encode(), b"TTF."+bytes(version)+b"."+parts[0]+b"."+parts[1], hashlib.sha256)
    t_hmac = base64.b64encode(t_hmac.digest())
    return f"{parts[0].decode('utf-8')}.{parts[1].decode('utf-8')}.{t_hmac.decode('utf-8')}"
```
 this particular token was genearted with these params 
```py
generate(9, "344404945359077377", "344404945359077377", 1502182937)
```

### CODE REVIEW

**Comment by Bluenix2** [comment]
> believable fake token 😉 based on your snowflake 😳 .

I actually want it to be un-believable, so that someone reading doesn't think it is real (and makes a PR to remove it). Can you maybe change it to `...` instead then?

> Also I assume you wanted to put this https://github.com/Bluenix2/wumpy/blob/main/library/wumpy/gateway/shard.py and not this https://github.com/Bluenix2/wumpy/library/wumpy-gateway/wumpy/gateway/shard.py ?

Yes, you can change that!

**Comment by arHSM** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ threading fashion or asyncio/trio/curio.
 ## Usage
 
 For a reference implementation see
-[wumpy-gateway](https://github.com/Bluenix2/wumpy/library/wumpy-gateway/wumpy/gateway/shard.py).
+[wumpy-gateway](https://github.com/Bluenix2/wumpy/blob/main/library/wumpy/gateway/shard.py).
```
Comment (Bluenix2) [inline README.md]: I don't think I read the link closely enough, it seems like it should be:
https://github.com/Bluenix2/wumpy/blob/main/library/wumpy-gateway/wumpy/gateway/shard.py

Comment (arHSM) [reply README.md]: Changed 👍

**Comment by Bluenix2** [comment]
Thanks for this!

**Pull request merged**